src/r_main.c

```diff
@@ -621,15 +603,50 @@ angle_t R_InterpolateAngle(angle_t oangle, angle_t nangle, fixed_t scale)
     }
 }
 
+static void R_SetupMouselook(fixed_t viewpitch)
```

Comment:
When changing the FOV, the view is glitched until moving the mouse. I'm not sure why.

Reply:
Fixed in https://github.com/fabiangreffrath/woof/pull/1389/commits/1a1695d86af5a42f596f69f066354d1b0dcc622a